SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -1506,6 +1506,32 @@ \section{Initialization and Assignment}
 \end{minted}
 \normalsize
 
+Nevertheless, there may be some exceptions to the initialization list rule. In
```

Comment:
The setting `DynamicMultiThreadingOn` is not an exception to the initialization rule -- this method is called in the constructor, but it is called from inherited classes, not from the class that has the `m_DynamicMultiThreading` member.  The member can only be initialized in an initialization list by the class of which it is a member. This is why we were getting those compile errors.

In general, when members are modified from inherited classes, they should use the `Set` and `Get` methods for the reasons listed below.

Reply:
:blush: True. d07be39 removes the `DynamicMultiThreadingOn` example.